asset/styles/style.css

```diff
@@ -0,0 +1,498 @@
+footer {
+  margin-left: 2%;
+  margin-right: 2%;
+  overflow-x: hidden;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+.header-logo {
+  height: 50px;
+}
+
+.header-logo-lg {
+  height: 60px;
+}
+
+.date-text {
+  color: #fff;
+  margin-left: 4%;
+}
+
+.header-main-container {
+  background-color: #f72210;
+}
+
+.login-text,
+#header-user-icon,
+#header-menu-bars {
+  color: #fff;
+  text-decoration: none;
+  font-weight: bold;
+}
```

Comment:
- [ ] You should still use the best practices in your CSS for this project, please avoid using ids to style items, prefer classes instead. Otherwise it's really easy to cause bugs and have extremely long files in your styles as the project grows due to how specific ids are and how you cannot re-use those styles.